pkg/cli/cmd/app/graph/graph.go

```diff
@@ -24,17 +24,17 @@ import (
 func NewCommand(factory framework.Factory) (*cobra.Command, framework.Runner) {
 	runner := NewRunner(factory)
 	cmd := &cobra.Command{
-		Use:     "connections",
-		Aliases: []string{"graph"},
-		Short:   "Shows the connections for an application.",
-		Long:    `Shows the connections for an application`,
+		Use:     "graph",
+		Aliases: []string{"connections"},
```

Comment:
do we want connections as alias ?

Reply:
Does not make sense to me but better to check with PM.

Reply:
Yes let's keep it to preserve backwards compatibility

Reply:
@AaronCrawfis wants to keep it around. Let's log an issue, we can decide later when to address it.

Reply:
https://github.com/radius-project/radius/issues/7262